script.skin.helper.service/plugin.py

```diff
@@ -46,8 +46,36 @@ def __init__(self):
             elif action == "LAUNCH":
                 xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
                 path = sys.argv[2].split("&path=")[1]
-                xbmc.executebuiltin("Action(Close)")
                 xbmc.executebuiltin(path)
+            elif action == "FOCUSANDCLICK":
+                #used as workaround to display local media in extendedinfo actorinfo
```

Comment:
On my extendedinfo-actor dialog I have added an extra container which displays the library items for the actor.
This "hack" is the onclick action if a user clicks a movie/show in the list, it will then cycle through the containers (provided by extendedinfo) and select the correct title so it opens up in extendedinfo. This was the only way to get the window stack not joking with me.

I think it makes more sense to just add this properly to the extendedinfo addon and add an extra container with the local library items. That way no weird hacks like this are needed.

If you feel something for this addition to extendedinfo I can subit a PR to you